manager/api/core/src/main/java/io/apiman/manager/api/core/config/ApiManagerConfig.java

```diff
@@ -335,6 +337,10 @@ public String getDataEncrypterType() {
         return getPrefixedProperties("apiman-manager.notifications.email.");
     }
 
+    public String getApimanManagerUiEndpoint() {
+        return config.getString(APIMAN_MANAGER_UI_ENDPOINT, "http://localhost:8080/apimanui/api-manager/");
```

Comment:
We should probably do something more intelligent than this

I guess we can minimally figure out the bind host or something?